WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="personal-records">
+  <h3 class="text-center"><%= t 'persons.show.personal_records' %></h3>
```

Comment:
Does this have Anthony Brooks's famous "odd message"? See https://github.com/jfly/worldcubeassociation.org/blob/a61e385dea976d4b9426af7038160027c8214815/webroot/results/includes/person_personal_records_current.php#L49 and https://www.worldcubeassociation.org/results/p.php?i=2008SEAR01.